docs/configmgmt.md

```diff
@@ -15,6 +15,10 @@ The [OVS commandline](https://openvstorage.gitbooks.io/administration/content/Ad
 
 ##### Specific ASD node related keys
 ```
+/ovs/alba/asdnodes/main = {"client_timeout": 20}
+        (This file does not exist by default. It can be created and manipulated in case one requires to change global asdnode settings.)
```

Comment:
Indent is not necessary. State what the default value is and what the impact of this key is